corehq/apps/cloudcare/static/cloudcare/js/form_entry/fullform-ui.js

```diff
@@ -575,6 +575,12 @@ hqDefine("cloudcare/js/form_entry/fullform-ui", function () {
             return self.error() === null && self.serverError() === null;
         };
 
+        self.mediaSrc = function (resourceType) {
+            // needs to be defined before calling getEntry()
```

Comment:
If this function were in its previous location, would tests fail? If not, is there a way you could add a test to `entrycontrols_full_spec.js` that would?